### PR TITLE
Avishay/swagger add params

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -612,20 +639,36 @@ definitions:
...
+        description: OpenShift cluster name
       openshiftVersion:
         type: string
         pattern: '^4\.\d$'
+        description: OpenShift cluster version
       baseDnsDomain:
         type: string
+      clusterNetworkCIDR:
+        type: string
+        description: IP address block from which Pod IPs are allocated
```
Comment (eranco74) [inline swagger.yaml]: Consider adding that it should be in CIDR notation and have an exampe: e.g.  10.128.0.0/14

**Comment by avishayt** [comment]
@eranco74 Please review again

File: swagger.yaml [hunk 2]
```diff
@@ -174,6 +174,31 @@ paths:
...
           description: Internal server error
 
+  /clusters/{clusterId}/downloads/kubeconfig:
+    get:
+      tags:
+        - inventory
+      summary: Download the kubeconfig file for the specified cluster
+      operationId: DownloadClusterKubeconfig
+      produces:
+        - text/x-yaml
```
Comment (filanov) [inline swagger.yaml:184]: i hope it supported - not all the formats supported by go - i'll let you know when i try to implement it

Comment (avishayt) [reply swagger.yaml:184]: Yep no problem

**Pull request closed without merging**